core/schema/modulesource.go

```diff
@@ -1305,6 +1309,40 @@ func (s *moduleSourceSchema) moduleSourceRepoRootPath(
 	return src.Git.RepoRootPath, nil
 }
 
+func (s *moduleSourceSchema) introspectionJSONFile(
+	ctx context.Context,
+	src dagql.Instance[*core.ModuleSource],
+	args struct {
+		IncludeSelf dagql.Optional[dagql.Boolean]
```

Comment:
The SDK (Elixir,Java,Php) should not set this pr set it to `false`?

Reply:
It depends, for client generation you will want to set it to true to generate self bindings.
For `Codegen` you should not for now since we do not support self call yet

Reply:
We keep pushing self-bindings but they're actually pretty important. They just need some investment to figure out the best solution. Some of it depends on the language, but the hope is that there's something that the engine can standardize on to make it easier for SDKs.